modules/ROOT/pages/Development/Cpp/setup.adoc

```diff
@@ -28,8 +28,8 @@ Most of the time, like in our modding case, there are two `targets`.
 One target is for compiling our `modules` compatible with the Unreal Editor,
 and the second target is for compiling our `modules` compatible with the final game.
 The configuration files for these modules will be located in the `/Source/YourModReference` folder of your mod.
-The target configurations follow the `<ModuleName>.Target.cs` naming pattern for the "shipping" configuration
-and the `<ModuleName>Editor.Target.cs` naming pattern for the "development editor" configuration.
+The target configurations follow the `YourModuleName.Target.cs` naming pattern for the "shipping" configuration
+and the `YourModuleNameEditor.Target.cs` naming pattern for the "development editor" configuration.
```

Comment:
The `.Target.cs` files aren't related to a mod, but to the project, looks like this was leftover from SML2 maybe